.github/workflows/publish.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
+        ruby-version: 2.6
```

Comment:
Was this change intentional?  I think we bumped this to `2.7` recently to address: https://github.com/medic/cht-android/issues/340